types.d.ts

```diff
@@ -3091,6 +3091,62 @@ export declare class ConformerGenerator {
   getPreviousConformerContribution(): number;
 }
 
+export interface IForceFieldMMFF94Options {
+  // TODO
```

Comment:
we can add options support later. it's tricky because we need to convert a JS object to a Java HashMap